tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -753,17 +787,11 @@ Test set:
     OLS loss: $ols_loss2")
 ```
 
-```julia; echo=false
-# Verify the loss on the test set.
-@assert vi_loss2 < 0.01
-@assert bayes_loss2 < 0.000001
-```
-
 ```julia
```

Comment:
````suggestion
```julia; echo=false
# Verify the loss on the test set.
@assert vi_loss2 < 0.01
@assert bayes_loss2 < 0.000001
```

```julia
````